images/source/centos-paas-sig-openshift-origin.repo

```diff
@@ -0,0 +1,5 @@
+[centos-paas-sig-openshift-origin]
+name = CentOS PaaS SIG Origin Repository
+baseurl = http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
+gpgcheck = 0
+disable = origin* tuned* ansible* atomic* docker* kompose* openshift*
```

Comment:
This is kind of ugly but there's no `enable =` --- @tdawson is there a better way to sandbox this repository (or the `future` one) so I can only really get `openvswitch` from it? I'm not super opposed to this approach, it just seems ugly.